### ISSUE TITLE
508-defect-2 [COGNITION]: SHOULD use a list for a list of options

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#7803]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Under the 4th step, Apply, in the subway list, for the "subheadings" By mail, Compensation, In person, and By fax:
These **should** be a list with headings so the screen reader may provide the context. The way this is read by a screen reader, it sounds like the Veteran must complete the form by mail, as the context of options is not read. Compensation isn't clear as the type of benefit applied for. I consider this serious, but not a launch blocker, as it is confusing for non-sighted users, and may lead to unnecessary extra effort to submit claims by having to leave the house.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I need to understand content with the context that is provided to sighted users through visual design.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: all
* Server destination: Devshop?/production

## Steps to Recreate

1. Enter https://preview-prod.vfs.va.gov/preview?nodeId=3011 in browser
2. Start screenreading device listed in Environment
3. Navigate to the fourth step by tabbing to the preceding link, and then pressing down arrow to read each line of content
4. Verify the lack of context

## Possible Fixes (optional)

- It might be better if, instead of "Compensation", it read "Address to send Compensation claims to" or add sr-only clarity as:
   ````html
         Compensation <span class="sr-only">claims should be mailed to:</span>
```    

- Additionally, perhaps state the three options in the preceding sentence, such as: 
    ```html
         Fill out the Decision Review Request: Supplemental Claim (VA Form 20-0995).          
         Link: Download VA Form 20-0995 (PDF)

         Submit your application by mail, in-person, or by fax.
```

## WCAG or Vendor Guidance (optional)

* [18f Accessibility Guide, Hidden Content](https://accessibility.18f.gov/hidden-content/)
* [Smashing mag, Web Accessibility in Context](https://www.smashingmagazine.com/2019/06/web-accessibility-context/)
* [24ways, Accessibility through Semantic HTML](https://24ways.org/2017/accessibility-through-semantic-html/)

**Comment by kelsonic** [comment]
Could we add a link to the URL where this error is coming up? The preview link doesn't work anymore 🙂

**Comment by jenstrickland** [comment]
Looks like this one is on Supplemental Claims and Board Appeals. 

I searched for 'options' on the spot check epic, https://github.com/department-of-veterans-affairs/va.gov-team/issues/7339.

**Comment by kelsonic** [comment]
Thank you! 🙂

**Comment by kelsonic** [comment]
Same thing here: https://github.com/department-of-veterans-affairs/va.gov-team/issues/7770

I can't update anything in `vets-website` that would resolve this

CC @stephorkandatx

**Comment by stephorkandatx** [comment]
@RLHecht please take a look into this. If we make modifications, let's record here and have Jen validate. If we investigate and determine to not make changes, we can use this ticket to alert Jen with our reasoning. Thank you!

**Comment by RLHecht** [comment]
@lwalsh617 As a fix for this, could we add the sentence that Jen is proposing above in step 4 of the subway map after the link to the Form PDF?
"Submit your application by mail, in-person, or by fax. FYSA @stephorkandatx

**Comment by RLHecht** [comment]
@lwalsh617 See my comment above and let me know what you think. Thanks!

**Comment by lwalsh617** [comment]
@RLHecht @stephorkandatx -- Just needed to look at it in context. I think this is a good solve. 
[submit 508 solve.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/4752802/submit.508.solve.docx)

**Comment by lwalsh617** [comment]
@RLHecht I made this edit to:
http://preview-prod.vfs.va.gov/preview?nodeId=3011
http://preview-prod.vfs.va.gov/preview?nodeId=3029
http://preview-prod.vfs.va.gov/preview?nodeId=3003
http://preview-prod.vfs.va.gov/preview?nodeId=3030

**Comment by RLHecht** [comment]
These have all been reviewed and published. I took out the hyphen in "in person" in all those subheads. Moving ticket to validate! FYSA @stephorkandatx

**Comment by jenstrickland** [comment]
Validated.
